src/main/java/com/marginallyclever/makelangelo/plotter/plottercontrols/MarlinInterfaceEvent.java

```diff
@@ -0,0 +1,39 @@
+package com.marginallyclever.makelangelo.plotter.plottercontrols;
+
+public class MarlinInterfaceEvent {
```

Comment:
you can transform this class to an `enum`

Reply:
an enum that contains a string and an int?  uh...

Reply:
Yes, an enum can contains multiples values associated in key / pair. This is exactly what is done in this class with attributes `id`, `command`. You can use this enum later in a switch case more easily and make sure no other value can be put in this object.